data/campaigns/tutorial/scenarios/01_Tutorial_part_1.cfg

```diff
@@ -121,7 +121,7 @@
 • <small>Healing</small>
 • <small>Recruiting</small>"
             [objective]
-                description= _ "Destroy a fierce enemy"
+                description= _ "Learn how to play"
```

Comment:
I think the original's better. Can we leave it as it is?

Reply:
I second this but didn't say anything because I didn't feel that strongly either way.